### ISSUE TITLE
Nightly Benchmarks for Bulk Sampling (PyG), Feature Fetching (PyG), and MFG Creation (PyG)

### ISSUE THREAD

**Issue by alexbarghi-nv** [rapidsai/cugraph#3637]
Add benchmarks for these 3 operations that will be run nightly.

Verification criteria:
1. Benchmarks are merged into cuGraph repository
2. Benchmarks successfully run nightly and report their results somewhere

**Comment by BradReesWork** [comment]
blocked - code is being refactored, waiting for that before this issue

**Comment by kingmesal** [comment]
If this is not unblocked by Sep. 8th, it will need to be prioritized for the following release.

**Comment by rlratzel** [comment]
closing as a duplicate of rapidsai/graph_dl#177
